ERROR: Service 'alice-controller' failed to build

I tried to run 'Aries Cloud Agent - Python (ACA-py) Demo Controllers' @ play-with-docker without luck:
```
git clone https://github.com/hyperledger/aries-acapy-controllers.git
cd aries-acapy-controllers/AliceFaberAcmeDemo
LEDGER_URL=http://dev.greenlight.bcovrin.vonx.io ./run_demo webstart
```
failed with
```
Step 9/14 : RUN npm run build
 ---> Running in 33fec9ccc10d

> alice-controller@0.0.0 build /app
> ng build

Node.js version v10.24.1 detected.
The Angular CLI requires a minimum Node.js version of either v12.14 or v14.15.

Please update your Node.js version or visit https://nodejs.org/ for additional instructions.

npm ERR! code ELIFECYCLE
npm ERR! errno 3
npm ERR! alice-controller@0.0.0 build: `ng build`
npm ERR! Exit status 3
npm ERR!
npm ERR! Failed at the alice-controller@0.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-10-10T16_14_59_428Z-debug.log
The command '/bin/sh -c npm run build' returned a non-zero code: 3
ERROR: Service 'alice-controller' failed to build : Build failed

```

--- a/AliceFaberAcmeDemo/controllers/alice-controller/Dockerfile
+++ b/AliceFaberAcmeDemo/controllers/alice-controller/Dockerfile
@@ -1,5 +1,5 @@
 # Stage: build
-FROM node:10 as build
+FROM node:14 as build
 WORKDIR /app
 COPY package*.json /app/
 RUN npm install

